File: Java/Java-main/J4/j4a.java
import java.util.Scanner;

public class j4a {
	public static void main(String[] args){
		Scanner jahre = new Scanner(System.in);
        double guthaben = 2000;
		double zinssatz = 1.03;
		System.out.print("Wie lange dauert die Verzinsung(in Jahren)?: ");
		int n = jahre.nextInt();
		
		for(int i = 1; i <= n; i++) {
            guthaben = guthaben * zinssatz;
            System.out.println("Nach " + i + " Jahren " + " erreichst du den Betrag: " + guthaben + " euro ");
		}
	}
}

File: Python/J4/j4a.py
guthaben = 2000
zinssatz = 1.03

while True:
    try:
        dauer = int(input("Wie lange soll die Verzinsung dauern? (In Jahren):\n"))
        for i in range(0, dauer):
            i += 1
            guthaben *= zinssatz
            if i == 1:
                print(f"Guthaben nach {i} Jahr: {round(guthaben, 2)} €")
            elif i > 1:
                print(f"Guthaben nach {i} Jahren: {round(guthaben, 2)} €")
        break
    except:
        print("Falsche Eingabe, versuche es erneut! \n")